testing/buildcop-function/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/GoogleCloudPlatform/cloud-run-samples/testing/buildcop-function
+
+go 1.14
+
+require github.com/googleapis/repo-automation-bots/packages/buildcop v0.0.0-20200716174820-aef5c4576904 // indirect
```

Comment:
Should this file be in the directory?

Reply:
The answer to this question and the one about go.sum in the script is the same:

Are we trying to pin a specific version of buildcop for download, or do we want to retrieve the latest version on each deployment? I'd argue we should "pin", in which case we should commit the go.mod and go.sum as part of managing this application.

Reply:
Added the go.sum!